src/main/resources/db/migration/V5.3.0.3__4475-disable-users.sql

```diff
@@ -0,0 +1,4 @@
+-- Users can be disabled.
```

Comment:
There is another script named V5.3.0.3 for expanded dataset field validation. How do we go about resolving? (Having deployed the other I'm getting a flyway script error when I try to deploy this branch.)

Reply:
I'm happy to let @scolapasta make a ruling.

Reply:
Since expanded dataset field validation is already in QA, let's change this one to be after.

Reply:
Ok, changed to be after in e73c43b70.